src/odemis/cli/main.py

```diff
@@ -53,6 +53,10 @@
 VAS_COMPS = {"alive", "dependencies"}
 VAS_HIDDEN = {"children", "affects"}
 
+# Command line arguments which can have "--" omitted
+ACTION_NAMES = ("kill", "check", "list", "list-prop", "set-attr", "update-metadata",
+                "move", "position", "stop", "reference", "acquire", "live", "scan")
```

Comment:
I know it's an optional argument, but what would be the side effect of including `help` as well?

Reply:
That's an oversight. "git help" works, so "odemis-cli help" would be logical that it works too. I've also added "version".